appveyor.yml

```diff
@@ -8,7 +8,7 @@ environment:
     - TOXENV: 'py27-t310-c45,py27-t40-c45,py27-t41-c45'
     - TOXENV: 'py34-t310-c45,py34-t40-c45,py34-t41-c45'
     - TOXENV: 'py35-t310-c45,py35-t40-c45,py35-t41-c45'
-    - TOXENV: 'pypy-t310-c45,pypy-t40-c45,pypy-t41-c45,pypy3-t310-c45,pypy3-t40-c45,pypy3-t41-c45'
+    - TOXENV: 'pypy-t310-c45,pypy-t40-c45,pypy-t41-c45'
```

Comment:
I think it could be kept, but the problem appears to be that only pypy gets installed (unconditionally)?!

Reply:
See https://github.com/pytest-dev/pytest/commit/ec69514eb2be687f751e3a7a18d96e3e29d44c16 for installing it only if needed.

Reply:
We don't install pypy there at all - it's preinstalled. Either way, I don't want to bother with pypy3 yet on Windows (it's beta anyway, and xdist is broken on pypy windows, so why bother with extensive testing on such a platform).